src/routes/index.js

```diff
@@ -40,59 +42,60 @@ baseRouter.get('/docs', swaggerUi.setup(swaggerDocument));
 // Auth
 versionOneRouter.post('/auth', authUser);
 versionOneRouter.post('/auth/check', checkUserToken);
+versionOneRouter.get('/auth/token', getToken);
```

Comment:
Since this is a response handler and nota util, can you move it from /utils and put it in its own routes file? routes/auth.js or something like that. You could even centrally locates this function and the user auth functions above into that file